src/components/ResultsItemDetails.vue

```diff
@@ -14,6 +14,11 @@
     </article>
     <button class='button-get-directions'>Get Directions</button>
     <!-- button is not functional yet - need to get a directions component w/ router -->
+    <section v-if="this.directions.length">
+      <section :key='i' v-for="(direction, i) in directions">
+        <h4>{{i+1}}. {{direction}}</h4>
```

Comment:
I appreciate this logic, so that the user doesn't start viewing directions at Step 0.

Reply:
I like this logic too, but I'm curious why you chose to do this over an ordered list?